doc/source/tutorial/interpolate/smoothing_splines.rst

```diff
@@ -372,6 +384,40 @@ a folium of Descartes *plus* some noise.
    >>> plt.show()
 
 
+Batching of `y` arrays
+----------------------
+
+Unlike :ref:`interpolating splines <tutorial-interpolate_batching>` and
+:ref:`GCV smoothers <tutorial-interpolate_GCV_batching>`, `make_splrep` and `make_splprep`
+do **not** allow multidimensional ``y`` arrays and require that ``x.ndim == y.ndim == 1``.
+
+The technical reason for this limitation is that the knot vector depends on the ``y``
+values, thus for a batched ``y``, the ``t`` batch could be a ragged array, which `BSpline`
+is not equipped to handle.
+
+Therefore if you need to handle batched inputs, you will need to loop over the
+batch manually and construct a `BSpline` object per slice of the batch.
+Having done that, you can mimic `BSpline` behavior for evaluations with a workaround
+along the lines of
+
+.. code-block:: python
+
+    class BatchSpline:
+        """BSpline-line class with reference batch behavior."""
```

Comment:
```suggestion
        """BSpline-like class with batch behavior."""
```
"reference" was used in the context of the tests, but probably isn't needed here.
(and "line" was a typo; I meant "like")